.github/workflows/contribution-counter.yml

```diff
@@ -0,0 +1,41 @@
+name: Contribution Counter
+
+on:
+  schedule:
```

Comment:
Why is this line `schedule` not commented out ? when does this action run if it has a `schedule` section with no data in it?

Reply:
Have uncommented the cron now. Had commented it out during my testing until all looked good.